### PR TITLE
Added Vim Cheat Sheet and also created a new Code Editors Folder.

### PR DESCRIPTION
**Pull request by Atharva8**
I have added Vim Cheat Sheet  and also updated the readme for the same. Thanks

### CODE REVIEW

**Comment by darkmatter18** [comment]
@Atharva8 Nice Work. But Please make a `Code Editors` directory and inside that add `Vim` directory, and add your cheatsheet. Thanks :1

**Comment by Atharva8** [comment]
Sure

**Comment by Atharva8** [comment]
I have done it. Thanks for the tip

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Fill Free to use the CheatSheets.
 - [JavaScript](./JavaScript)
 - [SQL](./SQL)
 - [Statistics](./Statistics)
+- [Vim](./Vim)
```
Comment (darkmatter18) [inline README.md:21]: Change it to `Code Editor` inside that, add `Vim`

**Review by darkmatter18 — CHANGES_REQUESTED** [review]
You missed the `README` file

**Comment by Atharva8** [comment]
I have updated the readme

File: README.md (hunk 1 above)
Comment (Atharva8) [reply README.md]: Done

**Review by darkmatter18 — APPROVED** [review]
Nice job @Atharva8

**Comment by darkmatter18** [comment]
@Atharva8 Thanks for your contribution. 🎉🎊 Please don't forget to Star ✨ the Repo

**Pull request merged**